compiler-rt/test/pfp/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set(PFP_TESTSUITES)
+set(PFP_TEST_DEPS ${SANITIZER_COMMON_LIT_TEST_DEPS} unwind lld compiler-rt)
+set(PFP_TEST_USE_LLD True)
```

Comment:
What is this for?

Reply:
It is used by ``config.use_lld = @PFP_TEST_USE_LLD@``.

Reply:
Then we can change that to `config.use_lld = True`

Reply:
Done